### DISCUSSION TITLE
在这里贴邮箱和订单的有收到验证码的吗？

### DISCUSSION THREAD

**Discussion by abortabort** [getlantern/lantern-forum-cn#422] [Q&A]
往 support@getlantern.org 发邮件根本没回复过呀。

**Comment by abortabort** [comment]


**Reply by MaxMKMax** [reply]
@abortabort 您好. 您需要验证码什么的可以直接在这里找我. 我来帮您直接对接负责人. 
其他用户有单独在帖子要求的我们基本上都解决验证码的问题了了.

最新版本也有了. 应该之前的问题也好了.
我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb
